src/actions/user.js

```diff
@@ -43,13 +43,18 @@ export function loadUserData() {
           dispatch(loadProjectWorkflows()),
         ])
       } else {
-        return Promise.all([
-          dispatch(loadUserAvatar()),
-          dispatch(loadUserProjects()),
-          dispatch(loadNotificationSettings()),
-          dispatch(loadSettings()),
-          dispatch(loadProjectWorkflows()),
-        ])
+        dispatch(getAuthUser()).then(() => {
+          return Promise.all([
+            dispatch(loadUserAvatar()),
+            dispatch(loadUserProjects()),
+            dispatch(loadNotificationSettings()),
+            dispatch(loadSettings()),
+            dispatch(loadProjectWorkflows()),
+          ])
+        }).catch(() => {
+          dispatch(setState('errorMessage', ''))
```

Comment:
Ah, ok. Seemed odd initially to do this, but if it is to clear out the error so UI is better then this makes sense.